x-pack/plugins/watcher/public/sections/watch_edit/components/watch_edit_execute_detail/watch_edit_execute_detail.html

```diff
@@ -71,16 +70,16 @@
     <div
       class="kuiInputNote kuiInputNote--danger"
       ng-show="!watchEditExecuteDetail.isAlternativeInputValid && !watchEditExecuteDetail.isAlternativeInputEmpty"
-    >
-      Invalid JSON
-    </div>
+      i18n-id="xpack.watcher.sections.watchEdit.executeDetail.inputOverrides.invalidJsonErrorMessage"
```

Comment:
I believe we need to use uppercase for abbreviations, => invalidJSONErrorMessage